Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM python:3-alpine
 ARG BUILD_DIR=/build
 
 # Add img
-RUN apk add img --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
+RUN apk add img docker --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
```

Comment:
Do you need the docker version from testing ?

Reply:
Not really, let me move it lower.